components/account/AccountMoreButton.vue

```diff
@@ -21,39 +23,63 @@ const { client } = $(useMasto())
 const useStarFavoriteIcon = usePreferences('useStarFavoriteIcon')
 const { share, isSupported: isShareSupported } = useShare()
 
+function recordEngagement(dataGlean) {
+  engagement.record({
+    ui_identifier: dataGlean,
+    mastodon_account_id: account.id,
+    mastodon_account_handle: account.acct,
+    ...engagementDetails[dataGlean],
+  })
+}
+
 function shareAccount() {
+  recordEngagement('profile.more.share_account')
   share({ url: location.href })
 }
 
 async function toggleReblogs() {
-  if (!relationship!.showingReblogs && await openConfirmDialog({
-    title: t('confirm.show_reblogs.title', [account.acct]),
-    confirm: t('confirm.show_reblogs.confirm'),
-    cancel: t('confirm.show_reblogs.cancel'),
-  }) !== 'confirm')
-    return
+  if (!relationship!.showingReblogs) {
+    if (await openConfirmDialog({
+      title: t('confirm.show_reblogs.title', [account.acct]),
+      confirm: t('confirm.show_reblogs.confirm'),
+      cancel: t('confirm.show_reblogs.cancel'),
+    }) !== 'confirm')
+      return
+    recordEngagement('profile.more.show_boosts')
+  }
+  else {
+    recordEngagement('profile.more.hide_boosts')
+  }
 
   const showingReblogs = !relationship?.showingReblogs
   relationship = await client.v1.accounts.follow(account.id, { reblogs: showingReblogs })
 }
 
 async function addUserNote() {
+  recordEngagement('profile.more.add_note')
   emit('addNote')
 }
 
 async function removeUserNote() {
+  recordEngagement('profile.more.remove_note')
+
   if (!relationship!.note || relationship!.note.length === 0)
     return
 
   const newNote = await client.v1.accounts.createNote(account.id, { comment: '' })
   relationship!.note = newNote.note
   emit('removeNote')
 }
+
+function report() {
+  recordEngagement('profile.more.report')
```

Comment:
@wtfluckey this one `profile.more.report` could be removed to be consistent with the other actions that open confirmation dialogues

Or also, could maybe rename it to `report_open`?